app/assets/javascripts/pageflow/editor/views/edit_meta_data_view.js

```diff
@@ -34,9 +34,16 @@ pageflow.EditMetaDataView = Backbone.Marionette.Layout.extend({
       });
 
       this.input('credits', pageflow.TextAreaInputView);
-      this.input('author', pageflow.TextInputView);
-      this.input('publisher', pageflow.TextInputView);
-      this.input('keywords', pageflow.TextInputView);
+
+      this.input('author', pageflow.TextInputView, {
+        placeholder: pageflow.theming.attributes.default_author
+      });
+      this.input('publisher', pageflow.TextInputView, {
+        placeholder: pageflow.theming.attributes.default_publisher
+      });
+      this.input('keywords', pageflow.TextInputView, {
+        placeholder: pageflow.theming.attributes.default_keywords
+      });
```

Comment:
Attributes rendered in the config seed are available as `pageflow.config...` inside the editor js.